db/migrate/20220307053344_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
```

Comment:
```suggestion
      t.string :email, null: false
```

FYI
バリデーションをつけるのが一般的です！（今回は tutorial に書いてある通り進めていただいても大丈夫です）